README-devs.md

```
@@ -0,0 +1,140 @@
+  <a href="https://pkg.go.dev/pkg.world.dev/world-engine/cardinal" target="_blank">
+    <img src="https://pkg.go.dev/badge/pkg.world.dev/world-engine/cardinal.svg" alt="Go Reference">
+  </a>
+  <a href="https://x.com/WorldEngineGG" target="_blank">
+    <img alt="Twitter Follow" src="https://img.shields.io/twitter/follow/WorldEngineGG">
+  </a>
+</div>
+
+## Info for World Engine Developers
+
+If you are looking for info for developing games using the World Engine, try:
+- the [README.md](./README.md)
```

Markdownlint

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)